types/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "semicolon": false,
+    "whitespace": false
+  }
+}
```

Comment:
could prettier be run on this file, to add the missing line feed at the end of file?